internal/pkg/service/stream/mapping/recordctx/utils.go

```diff
@@ -1,37 +1,27 @@
 package recordctx
 
 import (
-	"strings"
-
 	jsoniter "github.com/json-iterator/go"
 	"github.com/keboola/go-utils/pkg/orderedmap"
-	"github.com/umisama/go-regexpcache"
 
 	serviceError "github.com/keboola/keboola-as-code/internal/pkg/service/common/errors"
 	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/httputils"
 	utilsUrl "github.com/keboola/keboola-as-code/internal/pkg/utils/url"
 )
 
 // json - replacement of the standard encoding/json library, it is faster for larger responses.
 var json = jsoniter.ConfigCompatibleWithStandardLibrary //nolint:gochecknoglobals
 
-func isContentTypeJSON(t string) bool {
-	return regexpcache.MustCompile(`^application/([a-zA-Z0-9\.\-]+\+)?json$`).MatchString(t)
```

Comment:
Why can't we have strict mime types?

Reply:
or other point of view. Why it is not called `isApplicationType` so there is not `JSON` word in the name.

Reply:
```
.json	JSON format	application/json
.jsonld	JSON-LD format	application/ld+json
```
I see point only if this is the case

Reply:
@jachym-tousek-keboola regexp is there because clients (various net routers fo example) send other `Content-Types` when it is JSON and they cannot be configured.